.env

```diff
@@ -1,4 +1,5 @@
 API_URL="https://yayawallet.com/api/en"
```

Comment:
can you instead use `https://api.yayawallet.com/api/en` as this will be the production URL

Reply:
I might need some clarification on this. Taking it to slack.